packages/story-editor/src/app/pageDataUrls/pageDataUrlsProvider.js

```diff
@@ -52,7 +52,10 @@ function PageDataUrlProvider({ children }) {
         }));
       };
 
-      const clearQueueOfPageTask = queueIdleTask([idleTaskUid, idleTask]);
+      const clearQueueOfPageTask = queueIdleTask({
```

Comment:
Curious why the change?

Reply:
It makes the code easier to understand and type.